frontend-ssr/components/exercise-submissions/ExerciseSubmissionsPage.tsx

```diff
@@ -33,7 +33,6 @@ function ExerciseSubmissionPage(props: { submissions, dispatch }) {
       <br />
       <ul>
         {submissions.submissions.map((submission) => (
-          console.log(submissions),
```

Comment:
This commit should be squashed with the second one so we have just two in total.